### ISSUE TITLE
New rougelike dungeon messed generation

### ISSUE THREAD

**Issue by Kehaan** [Team-RTG/Realistic-Terrain-Generation#733]
Using newest Rougelike roguelike-1.7.10-1.5.0b i got this strange generation of one of the dungeons:
![2016-05-28_19 44 55](https://cloud.githubusercontent.com/assets/6755642/15628861/c72bc22c-250c-11e6-8a12-1e4dcdf2c673.png)
![2016-05-28_19 45 00](https://cloud.githubusercontent.com/assets/6755642/15628862/c7436012-250c-11e6-96a8-5c520a4886d6.png)

Log:
https://www.dropbox.com/s/x31may5eokcl4ca/fml-client-latest.log?dl=0

**Comment by srs-bsns** [comment]
Sorry, I don't really have free time to track this down right now.

**Comment by whichonespink44** [comment]
No worries mate - I'll see if I can reproduce it... gives me a chance to check out the new RLD :smile:

**Comment by whichonespink44** [comment]
This appears to be a bug on RLD's side when generating dungeons in Jungle biomes.

To confirm, I set `spawnFrequency=3` in roguelike.cfg and created an all-Jungle world using Lonely Biome (without RTG installed), and those vine-covered structures were everywhere.

@Greymerk, I checked the RLD issue tracker but couldn't see anything related to 'vines' in any of the open or closed issues. Any ideas?
